tests/test_integration.py

```diff
@@ -77,6 +77,7 @@ def setUpClass(cls):
 
     def test_update_schemas(self):
         for model in db.Model._decl_class_registry.values():
+            print(model)
```

Comment:
Oh yeah, I forgot it doesn't print to console unless it fails, which kind of had me scratching my head.  I'd advocate for keeping it then.